astronomy/sol_gravity_model.proto.txt

```diff
@@ -208,11 +349,92 @@ gravity_model {
+        column {order: 7, cos:  0.4407572012470000e-06, sin: -0.1775578673430000e-05}},
+      row {
+        degree: 8,
+        column {order: 0, cos:  0.1444404754944000e-06, sin:  0},
+        column {order: 1, cos: -0.1325390936354000e-06, sin:  0.7505455226430000e-06},
+        column {order: 2, cos:  0.1810373860406000e-05, sin:  0.5071656553342998e-06},
+        column {order: 3, cos: -0.1206916693583000e-05, sin: -0.1341432146034000e-05},
+        column {order: 4, cos:  0.1588293689104000e-05, sin:  0.1483187128519000e-06},
+        column {order: 5, cos: -0.2791513754339000e-05, sin: -0.1630200924016001e-05},
+        column {order: 6, cos: -0.9141050129427001e-06, sin: -0.1790077854570000e-05},
+        column {order: 7, cos: -0.4739785177040001e-06, sin:  0.1644602583282000e-05},
+        column {order: 8, cos: -0.3106010854034000e-06, sin: -0.2503541930098000e-06}},
+      row {
+        degree: 9,
+        column {order: 0, cos: -0.2873968549666000e-06, sin:  0},
+        column {order: 1, cos:  0.4209960192536999e-06, sin: -0.4901949692322000e-06},
+        column {order: 2, cos:  0.1138712441662000e-05, sin:  0.3822469142506001e-06},
+        column {order: 3, cos: -0.9935503979552999e-06, sin: -0.1007717243039000e-05},
+        column {order: 4, cos:  0.2948078161495000e-06, sin:  0.1623611577512000e-05},
+        column {order: 5, cos: -0.2271676246899000e-05, sin: -0.1549632021705000e-05},
+        column {order: 6, cos:  0.8084961361306998e-06, sin:  0.5778814559320999e-06},
+        column {order: 7, cos: -0.6181358420135002e-06, sin:  0.8687893503869002e-06},
+        column {order: 8, cos:  0.1208115622351000e-05, sin: -0.1461750901679999e-06},
+        column {order: 9, cos: -0.1171732975847000e-05, sin: -0.6580905885765000e-06}},
+      row {
+        degree:10,
+        column {order: 0, cos:  0.7268736890787999e-06, sin:  0},
+        column {order: 1, cos:  0.9239744729445000e-06, sin:  0.2232404729807000e-06},
+        column {order: 2, cos:  0.7099004701805000e-08, sin: -0.1115796893452000e-05},
+        column {order: 3, cos: -0.2990650877323000e-06, sin:  0.4435496951367000e-06},
+        column {order: 4, cos: -0.1213937691644000e-05, sin: -0.6987473706070001e-07},
+        column {order: 5, cos:  0.4215775932972000e-06, sin: -0.1062846147969000e-05},
+        column {order: 6, cos:  0.6694031372046999e-06, sin:  0.1117227455734000e-05},
+        column {order: 7, cos:  0.3179744736365000e-06, sin: -0.6201811340881001e-06},
+        column {order: 8, cos:  0.5404210955483000e-06, sin:  0.8206889582226001e-06},
+        column {order: 9, cos: -0.1457153463028000e-05, sin: -0.1463925755399000e-05},
+        column {order:10, cos: -0.2749907798463001e-06, sin:  0.7532015544117000e-06}},
+    }
   }
   body {
-    # Gravitational characteristics from MESSENGER data,
+    # Spherical harmonic coefficients and reference radius from MESSENGER data,
```

Comment:
Citation: Mazarico et al. (2014), *The gravity field, orientation, and ephemeris of Mercury from MESSENGER observations after three years in orbit*.

Note: while this is not obvious from the file name, this is HgM005.